### PR TITLE
@l2succes: [Structured Data] Add missing properties

### PR DESCRIPTION
**Pull request by xtina-starr**
This addresses https://artsyproduct.atlassian.net/browse/GROW-878 which wraps up the artwork structured data qa items. It's labeled as WIP because I'd like to add some tests.

- Uses `sale_message` to display the price.
- Breaks up price range into `minPrice` and `maxPrice` when applicable.
- Map availability to expected offer schema.
- Add missing brand property
- Type seller as "ArtGallery"

### CODE REVIEW

File: src/desktop/apps/artwork/helper.js [hunk 1]
```diff
@@ -1,15 +1,42 @@
-import { pickBy, identity } from "lodash"
+import { identity, map, pickBy, trim } from "lodash"
+import { toSentence } from "underscore.string"
 const { APP_URL } = require("sharify").data
 
+const AVAILABILITY = {
+  "for sale": "https://schema.org/InStock",
```
Comment (l2succes) [inline src/desktop/apps/artwork/helper.js]: are we sure these two are the only options for `availability`?

File: src/desktop/apps/artwork/helper.js [hunk 2]
```diff
@@ -1,15 +1,42 @@
...
+
 export const compactObject = o => {
   return pickBy(o, identity)
 }
 
+const displayPrice = artwork => {
+  const { is_price_hidden, is_price_range, price, sale_message } = artwork
+
+  if (is_price_range && !is_price_hidden) {
+    const minAndMaxPrice = price.split("-")
```
Comment (l2succes) [inline src/desktop/apps/artwork/helper.js]: add a check in case price is an `""` for instance

File: src/desktop/apps/artwork/helper.js [hunk 3]
```diff
@@ -1,15 +1,42 @@
...
+  }
+}
+
 export const convertArtworkToJSONLD = artwork => {
   let jsonLD
   const imageUrl =
     artwork.images && artwork.images.length ? artwork.images[0].url : ""
 
+  const artistsName =
+    artwork.artists.length && toSentence(map(artwork.artists, "name"))
```
Comment (l2succes) [inline src/desktop/apps/artwork/helper.js]: `artwork.artists && artwork.artists.length && ...`

File: src/desktop/apps/artwork/helper.js (hunk 1 above)
Comment (xtina-starr) [reply src/desktop/apps/artwork/helper.js]: It's for sure not the only one we support but not quite sure how we would reflect those in a way that matches the schema https://schema.org/ItemAvailability

File: src/desktop/apps/artwork/helper.js (hunk 2 above)
Comment (xtina-starr) [reply src/desktop/apps/artwork/helper.js]: Good catch!

File: src/desktop/apps/artwork/helper.js (hunk 1 above)
Comment (l2succes) [reply src/desktop/apps/artwork/helper.js]: Just make sure you have a fallback for states that don't map in this object. You can have any of these types based on schema.org. https://schema.org/ItemAvailability

- Discontinued
- InStock
- InStoreOnly
- LimitedAvailability
- OnlineOnly
- OutOfStock
- PreOrder
- PreSale
- SoldOut

Comment (xtina-starr) [reply src/desktop/apps/artwork/helper.js]: I talked to @junybrum and she mentioned that we wouldn't want to display the availability for artworks that aren't either `for_sale` or `sold`.

**Pull request merged**